### DISCUSSION TITLE
Integration not working - no sensor

### DISCUSSION THREAD

**Discussion by Peacem4kr** [Chouffy/home_assistant_tgtg#126] [Q&A] [answered]
Hi, 

i was following this [LINK](https://github.com/Chouffy/home_assistant_tgtg).
Created the Python file, got back the info i needed to add to my configuration.yaml file, restarted HA.
But now what?
It would be my understanding that there would be a sensor.tgtg, so i checked my entities list but can not find it.
How do i know that the integration is actually up and running?

1 thing though, i first copied the full info i received into the confirguration.yaml file, but my config.yaml did not want to save, my guess is that "sensor:" schould only exist once.
So i added this underneath.

![image](https://github.com/user-attachments/assets/131a95ce-dbcb-45f8-b1cc-567b6c689cf0)

Any idea what i can check or what i am doing wrong?
I tried to change the 'value' to "value", but this did not make any difference :)

Thank you!

**Comment by tjorim** [comment]
Anything in the logs?

**Comment by Peacem4kr** [comment]
Hi @tjorim , thx for the reply.
I dont think so:
![image](https://github.com/user-attachments/assets/d23a57ce-7153-4031-bccb-87c9a6c4d3f6)


First one is a camera which is not plugged in.
2 and 3 are warnings, but cant find info on tgtg in these.
4th one is a dog device tracker.

So i would say no.
Or do i need to check somewhere else?

I also tried to run the python script again to request a new token, but same issue.

Thank you!

**Reply by tjorim** [reply]
Just to be sure, do you have items marked as favourites?

**Reply by Peacem4kr** [reply]
i do not have any favorites, it was my understanding that this would be optional?
My end goal, perferably,  would be to receive a list of any new thing within 5km, like i would do in the app.

UPDATE: added some favorites in my app, restarted HA, now i have 2 sensors, 1 for each favorited item.

Thank you!

**Reply by Peacem4kr** [reply]
did this now favorite this 1 item? Or the shop? :)
Will the sensor update with future offers of this same shop?

**Reply by tjorim** [reply, marked as answer]
It's only the favourites, every surprise bag is separately, not a whole store. There is no location information to show nearby things. You can manually add items if you don't want to favourite them (see readme).

**Reply by Peacem4kr** [reply]
thank you for all the replies.
Do i understand it correctly that i always need to open my tgtg App, favorite some items, and only them they are displayed in HA?
If this is correct what would be a good use case/ example then please?

**Reply by tjorim** [reply]
Yes, I just favourite all things and use the template in the readme to show the ones available.
